src/cmds/mmap_test/mmap_test.c

```diff
@@ -0,0 +1,9 @@
+#include <stdio.h>
+#include <sys/mman.h>
+
+int main() {
+	int *a = mmap(NULL, 0, PROT_READ | PROT_WRITE, MAP_SHARED | MAP_ANONYMOUS, -1, 0);
+	if (a != NULL) printf("Allocated memory\n");
```

Comment:
Move _printf()_ to the next line.
Our code style:
https://github.com/embox/embox/wiki/Code-Style